frontend/packages/container-security/src/components/__tests__/summary.spec.tsx

```diff
@@ -0,0 +1,132 @@
+import * as React from 'react';
+import { shallow, ShallowWrapper } from 'enzyme';
+import * as _ from 'lodash';
+import { HealthState } from '@console/shared/src/components/dashboard/status-card/states';
+import { SecurityBreakdownPopup, securityHealthHandler } from '../summary';
+import { ImageManifestVuln } from '../../types';
+
+jest.mock('react-i18next', () => {
+  const reactI18next = require.requireActual('react-i18next');
+  return {
+    ...reactI18next,
+    useTranslation: () => ({ t: (key: string) => key }),
+  };
+});
+
+const highVuln: ImageManifestVuln = {
```

Comment:
Can we use `fakeVulnFor` data define in `bad-pods.ts`?

Reply:
done